wgsl/index.bs

```diff
@@ -7371,18 +7372,18 @@ allows them to naturally use values defined in the loop body.
 </div>
 * <1> The initialization is listed before the loop.
 
-<div class='example glsl' heading='GLSL Loop with continue'>
-  <xmp>
-    int a = 2;
-    int step = 1;
-    for (int i = 0; i < 4; i += step) {
+<div class='example wgsl function-scope' heading="for loop with continue">
+  <xmp highlight=wgsl>
+    var a: i32 = 2;
+    let step: i32 = 1;
+    for (var i: i32 = 0; i < 4; i += step) {
       if (i % 2 == 0) continue;
```

Comment:
To align with other example(s), this can be:

```
if i % 2 == 0 { continue; }
```

Thank you!

Reply:
done